### PR TITLE
Math floor

### PR DESCRIPTION
**Pull request by KnoxGameTeacher**
### Description

Closes issue #1608
Created a directory and term document for C# Math.Floor() in docs/content/c-sharp/concepts/math-functions/

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by KnoxGameTeacher** [comment]
I have added the suggested changes to the pull request.

**Review by caupolicandiaz — APPROVED** [review]
@KnoxGameTeacher Overall this looks good. This is ready for another review

File: content/c-sharp/concepts/math-functions/terms/floor/floor.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+Title: '.Floor()'
+Description: 'Math.Floor() rounds down and returns the largest whole integer less than or equal to the given number.'
```
Comment (yangc95) [inline content/c-sharp/concepts/math-functions/terms/floor/floor.md:3]:
```suggestion
Description: 'Rounds down and returns the largest whole integer less than or equal to the given number.'
```

File: content/c-sharp/concepts/math-functions/terms/floor/floor.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+**Math.Floor()** is a static method that always rounds down and returns the largest whole integer less than or equal to the given number.
+
+## Syntax
+
+```pseudo
+Math.Floor(x);
+```
+
+- The `Math.Floor()` method is used to find the largest whole integer which is less than or equal to the argument, `x`, passed in. The method accepts arguments in float and double format.
+- Code using the `Math.Floor()` method must have access to the `System` namespace.
```
Comment (yangc95) [inline content/c-sharp/concepts/math-functions/terms/floor/floor.md:25]:
```suggestion
- The `System` namespace is required to access this method.
```

File: content/c-sharp/concepts/math-functions/terms/floor/floor.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+**Math.Floor()** is a static method that always rounds down and returns the largest whole integer less than or equal to the given number.
+
+## Syntax
+
+```pseudo
+Math.Floor(x);
+```
+
+- The `Math.Floor()` method is used to find the largest whole integer which is less than or equal to the argument, `x`, passed in. The method accepts arguments in float and double format.
```
Comment (yangc95) [inline content/c-sharp/concepts/math-functions/terms/floor/floor.md:24]:
```suggestion
- The `Math.Floor()` method is used to find the largest whole integer which is less than or equal to the argument, `x`, that is passed in. The method accepts arguments in float and double format.
```

File: content/c-sharp/concepts/math-functions/terms/floor/floor.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+- Code using the `Math.Floor()` method must have access to the `System` namespace.
+
+## Example
+
+The following example uses `Math.Floor()` to return the largest whole number.
+
+```cs
+//Include the System namespace
```
Comment (yangc95) [inline content/c-sharp/concepts/math-functions/terms/floor/floor.md:32]:
```suggestion
// Include the System namespace
```

**Review by yangc95 — COMMENTED** [review]
@KnoxGameTeacher nice entry that's straight to the point! ❤️ I left some minor comments. Please tag me once all of them are resolved! TY

**Comment by KnoxGameTeacher** [comment]
@yangc95 I have resolved the suggested edits.

**Review by yangc95 — APPROVED** [review]
Thanks for the request! Looks good for merge now 😄

**Comment by yangc95** [comment]
Thanks for your persistence and this nice addition @KnoxGameTeacher! Here's the live link: https://www.codecademy.com/resources/docs/c-sharp/math-functions/sqrt

It looks like you're currently anonymous:
Screen Shot 2023-02-20 at 5 01 02 PM
If you'd like to become a public contributor, make sure to add your GitHub username to your Codecademy profile and set your visibility to "everyone." If after some substantial time has elapsed and you're still anonymous, log out of your Codecademy account and sign back in using the GitHub option.

Don't hesitate to reach out if you have any addition concerns!

cc: @caupolicandiaz @KTom101

**Pull request merged**